mmv1/products/privateca/Certificate.yaml

```diff
@@ -19,6 +19,10 @@ description: |
 
   ~> **Note:** The Certificate Authority that is referenced by this resource **must** be
   `tier = "ENTERPRISE"`
+references:
+  guides:
+    'Certificate Authority Service Overview': 'https://cloud.google.com/certificate-authority-service/docs/overview'
```

Comment:
this link seems to be returning a 404, could you update it? good catch on this by the way!